src/main/java/org/folio/services/instance/InstanceService.java

```diff
@@ -115,6 +115,13 @@ public Future<Response> createInstance(Instance entity) {
           // api client invoking this endpoint. The response is returned
           // a little earlier so the api client can continue its processing
           // while the domain event publish is satisfied.
+          .compose(response -> {
+            if (response.getEntity() instanceof Instance instanceResponse) {
+              instanceRepository.linkInstanceWithSubjectSourceAndType(instanceResponse);
```

Comment:
could you say please, should we insert/delete data in the "instance_subject_source", "instance_subject_type" tables upon instance update as well? For example, if during instance update some of subject is added with source/type that has not been specified yet in these new tables, or when instance sent for update does not contain subject with type/source that is already specified in the new tables.